internal/datastore/spanner/spanner.go

```diff
@@ -252,15 +285,15 @@ func (sd *spannerDatastore) ReadWriteTx(ctx context.Context, fn datastore.TxUser
 		}
 
 		return nil
-	})
+	}, spanner.TransactionOptions{TransactionTag: transactionTag})
```

Comment:
any specific limitations around tagging a transaction? e.g. number of them? How long do they stay around in the DB? Do we have an understanding of how it affects write throughput?

Should we check if `config.Metadata` is present outside of the RTW not to tag transactions unless requested by the customer? I assume tagging is not free, and not every Spanner user may be using transaction metadata feature.

Reply:
It shouldn't have any impact, since I believe Spanner just sets it to empty by default